scripts/should-e2e-run

```diff
@@ -0,0 +1,76 @@
+#!/bin/sh
```

Comment:
Could this be moved to `.circleci` directory instead to not mix CI and build related scripts ?

Reply:
No, I'd prefer to have things in the same visible location than reproduce the mess that is the Travis setup.

Reply:
And reproduce the same mess in `scripts` :/ ?

Reply:
I'll just assume you haven't looked at how travis is set up and that's why my comment is not understood.

Thanks!